docs/guides/admin/docs/workflowoperationhandlers/select-version-woh.md

```diff
@@ -17,6 +17,29 @@ There are currently two ways of specifying which version to select, which are **
 - A version number
 - A combination of *source-flavors* and *no-tag* (see below).
 
+### How to find the desired version of the media package?
+
+Every time a workflow runs a _snapshot_ operation, a new version of the media package is saved to the Asset Manager, starting with version 0.
+
+There isn't currently an easy way to list all the versions of a media package so here are some tips:
+
+1. Open the event details, go to the _Workflows_ tab, and count all _snapshot_ operations that were run until you reach the one you want. To get to the operation list, for each workflow, choose _Details,_ and then _Operations_ > _Details_.
```

Comment:
Is this actually reliable? For example, if I upload an event via the Admin UI, "version 0" seems to be created before the first workflow operation has a chance to run, and only contains a couple XML files, not the media files. Maybe there are more cases where Opencast creates new mp versions that the user needs to know about in order to reliable use this tip?

Reply:
Probably. I will remove this because it isn't 100% reliable. I will leave only the database part. Or please let me know if you have other suggestions.

Reply:
Okay, makes sense. Sadly I don't have any other suggestions.